src/config.xml

```diff
@@ -88,6 +88,17 @@ For lists, items can also be appended using:
       TAG += value [value, ...]
 \endverbatim
 Values that contain spaces should be placed between quotes (\" \").
+<br>
+Note:<br>
+Use doxygen to compare the used configuration file with the template configuration file:
+\verbatim
+      doxygen -x [configFile]
+\endverbatim
+Use doxygen to compare the used configuration file with the template configuration file
+without replacing the environment variables>
```

Comment:
the > after variables should be rather a  ":" ?

Reply:
Thanks.
I've just pushed a proposed patch, pull request #9268